docs/datatype/image/text2image.md

```diff
@@ -1,3 +1,484 @@
 # Search Image from Text via CLIP model
 
-TBA
\ No newline at end of file
+In this tutorial, we will create an image search system that retrieves images based on short text descriptions of their content.
+
+The interest behind this is that regular search or queries rely on image description or meta data to describe the content of the image, this is simply not efficient 
```

Comment:
- meta data -> metadata
- of the image, this is simply... -> of the image. This can be expensive and is not efficient...